index.js

```diff
@@ -0,0 +1,142 @@
+import addToLocalStorage from './modules/localStorage.js';
+import time from './modules/dateTime.js';
+
+const bookInput = document.querySelector('.book-input');
+const authorInput = document.querySelector('.author-input');
+const submit = document.querySelector('.add');
+const booksDiv = document.querySelector('.books');
+
+let arrayOfBooks = [];
+
+if (localStorage.getItem('books')) {
+  arrayOfBooks = JSON.parse(localStorage.getItem('books'));
+}
+
+const addElementsToPage = ((arrayOfBooks) => {
+  booksDiv.innerHTML = '';
+
+  arrayOfBooks.forEach((task) => {
+    const contactElement = ' By : ';
+    const div = document.createElement('div');
+    div.className = 'task';
+
+    div.setAttribute('data-id', task.id);
+    div.appendChild(
+      document.createTextNode(task.title + contactElement + task.author),
+    );
+
+    const span = document.createElement('span');
+    span.className = 'del';
+    span.appendChild(document.createTextNode('Delet'));
+
+    div.appendChild(span);
+
+    booksDiv.appendChild(div);
+  });
+});
+
+const getFromLocalStorage = () => {
+  const data = window.localStorage.getItem('books');
+  if (data) {
+    const books = JSON.parse(data);
+    addElementsToPage(books);
+  }
+};
+
+// Trigger Get Data From Local Storage Function
+getFromLocalStorage();
+
+class AddingBooks {
```

Comment:
- [ ] [optional] It would have been nice if you had a module for adding books, it would make your code shorter and more readable.